packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Timeline/Reflow/SimplePanning.1.doc.mdx

```diff
@@ -0,0 +1,39 @@
+import TimelinePanning from '@cerner/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/Timeline/Reflow/SimplePanning.jsx';
+import PropsTable from '@cerner/terra-graphs/lib/components/Timeline/Timeline?dev-site-props-table';
+import GraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Timeline/timelineConfigObject?dev-site-codeblock';
```

Comment:
It looks like you used different config data for example.

Reply:
Updated here: 5f101bc17a20c4113abb0d38194407e62c01757f